app/src/main/java/com/zeapo/pwdstore/utils/FileUtils.kt

```diff
@@ -0,0 +1,133 @@
+package com.zeapo.pwdstore.utils
+
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.FileOutputStream
+import java.io.IOException
+import java.io.InputStream
+import java.nio.charset.Charset
+
+class FileUtils {
+    companion object {
+
+        @JvmStatic
+        fun listFiles(dir: File, extensions: Array<String>?, recursive: Boolean): Collection<File> {
+            val res = ArrayList<File>()
+            val files = dir.listFiles()
+
+            if (files != null && files.isNotEmpty()) {
+
+                files.forEach { file ->
+                    // Check if the file is a directory and recursive add
+                    if (file.isDirectory && recursive) {
+                        res.addAll(listFiles(file, extensions, recursive))
+                    } else if (!file.isDirectory) {
+                        if (extensions == null) {
+                            //not check extension
+                            res.add(file)
+                        } else if (extensions.contains(file.extension)) {
+                            res.add(file)
+                        }
+                    }
+                }
+            }
+
+            return res
+        }
+
+        @JvmStatic
+        fun readFileToString(file: File, charset: Charset): String {
+            return file.readText(charset)
+        }
+
+        @JvmStatic
+        @Throws(IOException::class)
+        fun openInputStream(file: File): FileInputStream {
```

Comment:
This should get replaced in all call sites with the stdlib `inputStream()` method.